src/main/java/com/petros/bibernate/config/properties/ReadPropertiesFromPropImpl.java

```diff
@@ -0,0 +1,20 @@
+package com.petros.bibernate.config.properties;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ReadPropertiesFromPropImpl implements ReadProperties {
```

Comment:
This class looks like redundant. In my opinion, it is good idea to move properties loading to Configuration class. It will follow single responsibility principle, ConfigurationImpl will be responsible for loading, storing and providing properties.

Reply:
What if you want to load properties of a different file type? e.g. yaml or xml

Reply:
We could use strategy pattern. But it is ok to leave current implementation. I'll approve, but could you rename ReadPropertiesFromPropImpl? It is a bad practice to use verbs in class names, we can rename it to PropertiesFileReader or PropertiesFileLoader, or whatever you want but with noun in name